src/shogun/distance/AttenuatedEuclideanDistance.h

```diff
@@ -17,7 +17,9 @@
 #include <shogun/distance/RealDistance.h>
 #include <shogun/features/DenseFeatures.h>
 
-namespace shogun
+using namespace shogun;
```

Comment:
I mean we better have namespace `shogun::distance` but currently it is just `distance`